test/sync/src/androidTest/kotlin/io/realm/test/mongodb/shared/AppTests.kt

```diff
@@ -37,7 +38,7 @@ class AppTests {
 
     @BeforeTest
     fun setup() {
-        app = TestApp()
+        app = TestApp(fileSystem = FileSystem.SYSTEM)
```

Comment:
It seems needlessly complicated to have to inject this everywhere? Why not just set the default value in the constructor?

Reply:
Not quite. `FileSystem.SYSTEM` is platform-specific so it cannot be referenced from `commonMain` directly and `TestApp` is placed there. I will write an `expect` variable and use it in the constructor and implement it for each platform instead.